lib/templates/test-body-footer.html

```diff
@@ -36,8 +36,35 @@
     });
   }
 
+  function getModuleName (){
+    return jQuery.ajax({
```

Comment:
I'd prefer not to rely directly on jQuery, if ember-cli includes an ajax lib let's use that - feel free to tell me that it's jQuery ;)